### ISSUE TITLE
gradient flow through the last value?

### ISSUE THREAD

**Issue by rizar** [lcswillems/pytorch-a2c-ppo#6]
I think that because of [this line](https://github.com/lcswillems/pytorch-a2c-ppo/blob/master/torch_rl/torch_rl/algos/base.py#L176) it is possible that the gradient will flow through the advantage estimate, which I guess is undesirable.

**Comment by lcswillems** [comment]
This code is inside `with torch.no_grad():` for there is no gradient in this part.

**Comment by rizar** [comment]
Right!
